README.md

```diff
@@ -64,7 +64,11 @@ Now you can proceed to any of the following tasks:
    ```console
      # cp roles/sat6repro/vars/main.sample.yml roles/sat6repro/vars/main.yml
    ```
-3. Update the [inventory] (inventory) file under the root of this project to add the IP address of the blank host you are cloning to. If executing the playbook on localhost, add `ansible_connection=local` after the IP address.
+3. Create an `inventory` file by copying `inventory.sample` in the root of this project. Add the IP address of the Destination node. If executing the playbook on localhost, add `ansible_connection=local` after the IP address.
+
+   ```console
+     # cp inventory.sample inventory
+   ```
```

Comment:
You don't need this because https://github.com/RedHatSatellite/satellite-clone/blob/master/README.md#getting-started is supposed to be common set of instructions for any playbook in this project and it already tells you to create a copy of the inventory.